cardano-db/src/Cardano/Db/Query.hs

```diff
@@ -65,6 +65,7 @@ module Cardano.Db.Query
   , unValueSumAda
   ) where
 
+import           Cardano.Prelude hiding (Meta, from, isJust, isNothing, maybeToEither, on)
```

Comment:
This change does not offer a net benefit.

Reply:
https://github.com/input-output-hk/cardano-db-sync/pull/527#discussion_r575045248